changelog.md

```diff
@@ -0,0 +1,62 @@
+This list is a collection of breaking changes:
```

Comment:
I really do appreciate the effort of tracking changes. The project does use conventional commits, so given the right format, that file is generated automatically for each github release: https://www.conventionalcommits.org/en/v1.0.0/

However, I think in this case a more concise "migration guide" would be helpful indeed, which we should place under the `docs/` folder IMHO, and maybe link from the main index file.